.circleci/config.yml

```
@@ -31,7 +31,7 @@ jobs:
                   conda update --yes --quiet conda
                   conda env create -f environment.yml --name=testenv
                   source activate testenv
-                  pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/master
+                  pip install mne==0.19
```

```suggestion
                  pip install mne>=0.19
```

you should be using 0.19.2 ie minor bug fix releases

---

done, I am using `mne~=0.19.0` now, [which will always install the highest bugfix release](https://www.python.org/dev/peps/pep-0440/#compatible-release)